packages/web/app/src/lib/preflight-sandbox/lab-api-declaration.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable  @typescript-eslint/no-unused-vars */
+
+// The content of this file is used as a string
+// and feed into the context of the Monaco Editor and the Preflight Script.
+// The lack of `declare const lab: LabAPI` is intentional, to avoid messing up the global scope
+// of the web app codebase.
+// This could be a string in `graphiql-plugin.tsx`, but it's better to keep it in a separate file,
+// and use Prettier to format it, have syntax highlighting, etc.
+
+interface LabAPI {
+  /**
+   * [CryptoJS](https://cryptojs.gitbook.io/docs) library.
+   */
+  CryptoJS: any;
```

Comment:
I assume there was no straight-forward way to load `@types/crypto-js` as a string? 😄 https://www.npmjs.com/package/@types/crypto-js?activeTab=code

Reply:
Too much hassle